src/DisplayEmailComponents/DisplayEmailCard/EmailCard.js

```diff
@@ -0,0 +1,16 @@
+function EmailCard(props) {
```

Comment:
some containers are written as a named function whereas others are written as an arrow function:
```js
function EmailCard(props) {....}

const DisplayEmailContainer = (props) => {....}
```

you should make it consistently one way or the other for each container